packages/kit-headless/src/components/tooltip/tooltip-panel.tsx

```diff
@@ -0,0 +1,23 @@
+import { PropsOf, Slot, component$, useContext } from '@builder.io/qwik';
+import { HPopoverPanel } from '../popover/popover-panel';
+import { TooltipContextId } from './tooltip-context';
+
+export type HTooltipPanelProps = PropsOf<typeof HPopoverPanel>;
+
+/**
+ * HTooltipPanel is the panel component for the Tooltip.
+ */
+export const HTooltipPanel = component$((props: HTooltipPanelProps) => {
+  const context = useContext(TooltipContextId);
+
+  return (
+    <HPopoverPanel
+      {...props}
+      role="tooltip"
+      id={context.localId}
+      data-state={context.state.value}
```

Comment:
There are 4 possible states for this component. Do we have individual `data-` fields for each or a single state field?

Reply:
answered in DM. happy to paste here if appropriate